docs/source/releases/v1.6.rst

```diff
@@ -36,6 +36,9 @@ Removal of deprecated features
 
 Minor changes
 ~~~~~~~~~~~~~
+ - Majority of the Oscar imports now using dynamic loading, instead of
+   direct importing for better extensibility and customizability, except
+   signals.
```

Comment:
I think it's worth adding a note to the backwards incompatible changes to say that signals are no longer dynamically loaded.

Reply:
Ok added.